site/source/docs/compiling/WebAssembly.rst

```diff
@@ -59,19 +59,17 @@ upgrade from fastcomp to upstream:
     codegen. The simple and safe thing is to pass all ``-s`` flags at both
     compile and link time.
 
-  * You can enable LTO object files with the usual llvm compiler flags (-flto,
-    -flto=full, -flto=thin, -emit-llvm).  These flags will make the wasm backend
-    behave more like fastcomp. With fastcomp LTO optimization passes will not
-    be run by default; for that you must pass ``--llvm-lto 1``.  With the llvm
-    backend LTO passes will be run on any object files that are in bitcode
-    format.
+  * You can enable Link Time Optimization (LTO) with the usual llvm flags
+    (``-flto``, ``-flto=full``, ``-flto=thin``, at both compile and link times).
+    These flags will make the wasm backend behave more like fastcomp.
+
+  * With fastcomp LTO optimization passes will not be run by default;
+    for that you must pass ``--llvm-lto 1``.  With the llvm backend
+    LTO passes will be run on any object files that are in bitcode format.
```

Comment:
I'm not sure what's the differences between `-flto` family of flags and `--llvm-lto 1`? What are the users supposed to use?

Reply:
The former is for the llvm backend only.  The later is for fastcomp only.  The effect is not identical because the two compilers do this differently, but they can be thought of as roughly equivalent.

`--llvm-lto` will die completely when fastcomp dies.